src/models/MainGoal.js

```diff
@@ -0,0 +1,82 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+
+const TodosSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    default: "",
+  },
+  addedInCalender: {
+    type: Map,
+    of: new mongoose.Schema(
+      {
+        isComplete: { type: Boolean, default: false },
+        memo: { type: String, default: "" },
+      },
+      {
+        _id: false,
+      },
+    ),
+  },
+  level: {
+    type: Number,
+    default: 0,
+  },
+  experiencePoint: {
+    type: Number,
+    default: 0,
+  },
+});
+
+const SubGoalSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    default: "",
+  },
+  todos: {
+    type: [TodosSchema],
```

Comment:
저같은 경우, 스키마에서 type이 array 일 때, default를 생락하고 아래처럼 사용하고는 있습니다!
```
todos: [TodosSchema]
```

[schematypes_array](https://mongoosejs.com/docs/schematypes.html#arrays) 보시면 
Arrays are special because they implicitly have a default value of [] (empty array) 라고 명시되어있어서요!

Reply:
아 그러네요! 말씀해주신 부분 수정해서 다시 올리도록 하겠습니다.